grafana-cloud-metrics.yaml

```diff
@@ -229,6 +232,13 @@ resources:
                     url: 'http://'+window.location.hostname+'/elasticsearch',
                     index: 'grafana-dash',
                     grafanaDB: true,
+                  },
+                  blueflood: {
+                    type: 'BluefloodDatasource',
+                    url: 'http://iad.metrics.api.rackspacecloud.com',
```

Comment:
But it will still work if the stack is deployed in a DC other than IAD, correct?